sentences/nl/assist_satellite_HassBroadcast.yaml

```diff
@@ -5,3 +5,5 @@ intents:
       - sentences:
           - "broadcast [overal] [dat] {message}"
           - "roep [overal] ([dat] {message};om)"
+          - "roep om [in het huis] {message}"
```

Comment:
if you add an optional `[dat]` you can say
`roep om in het huis dat we gaan eten` and it will broadcast `we gaan eten`

It makes the sentence more natural, and the `dat` part won't be included in the broadcast message. Without the optional `dat` it will broadcast `dat we gaan eten`

Reply:
You can combine the 2nd and 3rd one in
`- roep [overal|in het huis] ([dat] {message};om)`

That will also match
- roep in het huis [dat] {message} om
- roep in het huis om [dat] {message}

Reply:
on reading this again, we also need
`- roep om in het huis [dat] {message}`
with `overal`  the word `om` will never be used at that place in the sentence (_roep om overal dat we gaan eten_ makes no sense)

So I would say this covers all cases, including the ones suggested in this PR
```yaml
      - sentences:
          - "(broadcast|intercom) [overal|in het huis] [dat] {message}"
          - "roep [overal|in het huis] ([dat] {message};om)"
          - "roep om in het huis [dat] {message}"
```